### PR TITLE
feat: added modal

### PR DESCRIPTION
**Pull request by harshalini**
Added modal container which has two action buttons, on clicking the click me button, the dialog box appears.
Check it here ->
https://hopeful-lewin-ca6ba4.netlify.app/documentation/documentation.html
Screenshot (241)
Screenshot (242)

### CODE REVIEW

**Review by ShreyashMahajan — APPROVED** [review]
code looks very good and clean. UI is also very good

File: components/modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+.modal-content {
+    padding: 1rem 1.5rem 0 1.5rem;
+    background-color: var(--white);
+    color: black;
+    visibility: hidden;
+    margin-top: -6rem;
+}
+.modal-header {
+    font-size: 1.4rem;
+    border-bottom: solid 2px darkgray;
```
Comment (t007rushi) [inline components/modal/modal.css]: Color variables missing here only

**Review by t007rushi — CHANGES_REQUESTED** [review]
Small change to push for consistency

**Review by SaurabhChawda — APPROVED** [review]
Everything is fine.

File: components/modal/modal.css (hunk 1 above)
Comment (harshalini) [reply components/modal/modal.css]: Thanks for pointing out that, changed it to color variable, please approve the changes

**Comment by harshalini** [comment]
> code looks very good and clean. UI is also very good

Thanks!

**Review by t007rushi — APPROVED** [review]

**Pull request merged**